pkg/agent/prommetrics.go

```diff
@@ -48,6 +66,11 @@ const (
 	runnerMetricStatePanicked runnerMetricState = "panicked"
 )
 
+// Copied bucket values from controller runtime latency metric. We can
+// adjust them in the future if needed.
+var buckets = []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0,
+	1.25, 1.5, 1.75, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60}
```

Comment:
That's a ton of buckets. Can we get some data from staging first about how often this typically takes?